version.gradle

```diff
@@ -25,7 +25,7 @@
  *  as we want to manage the versions in a single source.
  */
 
-def final SPINE_VERSION = '0.11.00-SNAPSHOT'
+def final SPINE_VERSION = '0.11.01-SNAPSHOT'
```

Comment:
Please make it with no zero.  The next version would be `0.11.1-SNAPSHOT`. And if Dima already merged to `master`, this changeset should be an increment (e.g. `0.11.2-SNAPSHOT — please recheck).